colour/colorimetry/spectrum.py

```diff
@@ -3690,9 +3939,10 @@ def interpolate(self, shape=SpectralShape(), method=None):
 
     def align(self,
               shape,
-              method='Constant',
-              left=None,
-              right=None):
+              interpolation_method=None,
+              extrapolation_method='Constant',
+              extrapolation_left=None,
```

Comment:
Not sure about this method signature.
What is the meaning for `extrapolation_left` for  linear interpolation?